plugin/src/main/java/jenkins/plugins/openstack/compute/JCloudsCloud.java

```diff
@@ -164,6 +164,8 @@ private Object readResolve() {
         LauncherFactory lf = null;
         if ("JNLP".equals(slaveOptions.slaveType)) {
             lf = LauncherFactory.JNLP.JNLP;
+        } else if ("KAFKA".equals(slaveOptions.slaveType)) {
```

Comment:
There is no need to add these into `readResolve` methods. They are only used to migrate old config to new one. And since kafka is just being added, it does not exist in user configs.